tests/datasource/fluent/test_spark_datasource.py

```diff
@@ -43,7 +44,13 @@ def test_dataframe_asset(
     errors_dict = exc_info.value.errors()[0]
     assert errors_dict["loc"][0] == "dataframe"
 
-    datasource = empty_data_context.sources.add_spark(name="my_spark_datasource")
+    datasource_name = "my_spark_datasource"
+    datasource = empty_data_context.sources.add_spark(
+        name=datasource_name,
+        persist=False,
+        force_reuse_spark_context=True,
+        spark_config={},
+    )
```

Comment:
I think this might fit better in the `test_spark_config_passed_to_execution_engine` test.
Or maybe that test should be parametrized to include the `persist` key.

Reply:
Done